dbt_subprojects/dex/models/_projects/zeroex/arbitrum/zeroex_arbitrum_settler_addresses.sql

```diff
@@ -2,9 +2,15 @@
     config(
         schema = 'zeroex',
         alias = 'arbitrum_settler_addresses',
-        materialized = 'table'
+        materialized='incremental',
+        partition_by = ['begin_block_time'],
+        unique_key = ['token_id', 'settler_address', 'begin_block_number', 'begin_block_time'],
+        on_schema_change='sync_all_columns',
+        file_format ='delta',
+        incremental_strategy='merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')]
```

Comment:
```suggestion
```

Reply:
lets remove this since spell is small and we can refer to all of history for lookup to target